src/status_im/contexts/profile/push_notifications/events.cljs

```diff
@@ -145,8 +145,7 @@
                news-notifications-enabled?      (conj :disable-news-notifications?)
                messenger-notifications-enabled? (conj :disable-chat-notifications?))]
             :else nil)
-          (when enabled?
-            (save-profile-setting-fx permission-setting))
+          (save-profile-setting-fx permission-setting)
```

Comment:
Fixing a small bug here where we were not saving the setting when it was enabled